tutorminio/templates/minio/tasks/minio/init.sh

```diff
@@ -1,9 +1,36 @@
+# Verify that `MINIO_GCS_APPLICATION_CREDENTIALS` and `MINIO_GCS_APPLICATION_ID` are defined if `MINIO_GATEWAY == "gcs"`
+{% if MINIO_GATEWAY == "gcs" %}
+  {% if not MINIO_GCS_APPLICATION_CREDENTIALS %}
+    echo "⚠️  MINIO_GCS_APPLICATION_CREDENTIALS is not defined. It is required to when MINIO_GATEWAY is set to gcs."
```

Comment:
There is some funny character at the beginning of the the string ... not serious, but just in case.

Reply:
Same for the one below.